frappe/website/utils.py

```diff
@@ -80,8 +97,8 @@ def _get_home_page():
 def is_signup_enabled():
 	if getattr(frappe.local, "is_signup_enabled", None) is None:
 		frappe.local.is_signup_enabled = True
-		if frappe.utils.cint(frappe.db.get_value("Website Settings",
-			"Website Settings", "disable_signup")):
+		if frappe.utils.cint(frappe.db.get_value('Website', get_website_name(),
```

Comment:
why not just `get_website_settings('disable_signup')`

Reply:
`get_website_settings` could be confusing, since it used to be a DocType. Settled on `get_website_conf`. Looks much cleaner, thanks.

Reply:
Please use `get_website_settings` since it was Website Settings earlier. Also conf is for config files.

Reply:
okay, noted. :+1: